packages/website/components/button.js

```diff
@@ -66,12 +66,12 @@ export default function Button({
     [tracking, onClick, href]
   )
 
-  let btnClasses = clsx('btn button-reset select-none black', className)
+  let btnClasses = clsx('btn button-reset select-none color-black', className)
```

Comment:
**question:** What is "color-black" doing?

Reply:
nothing, should be text-black.